.pre-commit-config.yaml

```diff
@@ -20,28 +20,9 @@ repos:
      - id: trailing-whitespace
        args: [--markdown-linebreak-ext=md]
 
-  - repo: https://github.com/ikamensh/flynt/
-    rev: '1.0.1'
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.11.8
     hooks:
-     - id: flynt
-       args: [--line-length=79, --transform-concats]
-
-  - repo: https://github.com/psf/black
-    rev: 25.1.0
-    hooks:
-     - id: black
-
-  - repo: https://github.com/PyCQA/isort
-    rev: 6.0.0
-    hooks:
-     - id: isort
-       name: isort (python)
-
-#
-# Tried and Failed
-#
-# 1. I tried doing mypy too, but it was a mess because it runs in its own
-#   isolated environment, that lacks all our dependencies.
-# 2. We might want flake8 someday. It's easy to turn on, but it's so noisy on
-#    our current code that it'd take some fine-tuning to get it right.
-# 3. semgrep was too slow and had to be moved to a github action.
```

Comment:
I think this comment was copypasta from courtlistener:

https://github.com/freelawproject/courtlistener/blob/2a60b555b3b48896da02effa9bb0be806099c500/.pre-commit-config.yaml#L46-L53